ambry-tools/src/main/java/com.github.ambry/account/AccountUpdateTool.java

```diff
@@ -0,0 +1,171 @@
+ *        "status": "ACTIVE"
+ *      },
+ *      {
+ *        "accountId": 2,
+ *        "accountName": "account1",
+ *        "containers": [
+ *          {
+ *            "parentAccountId": 2,
+ *            "containerName": "container1",
+ *            "description": "This is the first container of ",
+ *            "isPrivate": true,
+ *            "containerId": 1,
+ *            "version": 1,
+ *            "status": "INACTIVE"
+ *          }
+ *        ],
+ *        "version": 1,
+ *        "status": "ACTIVE"
+ *      }
+ *     ]
+ *   </pre>
+ * </p>
+ */
+public class AccountUpdateTool {
+
+  /**
+   * @param args takes in three mandatory arguments: the path of the json file for the accounts to create/update,
+   *             the address of the {@code ZooKeeper} server, and the root path for the {@link org.apache.helix.store.HelixPropertyStore}
+   *             that will be used for storing account metadata and notifications.
+   *
+   *             Also takes in an optional argument that specifies the timeout in millisecond to connect to the
+   *             {@code ZooKeeper} server (default value is 5000), and the timeout in millisecond to keep a session
+   *             to the {@code ZooKeeper} (default value is 20000).
+   */
+  public static void main(String args[]) throws Exception {
+    long startTime = System.currentTimeMillis();
+    OptionParser parser = new OptionParser();
+
+    ArgumentAcceptingOptionSpec<String> accountJsonFilePathOpt = parser.accepts("accountJsonPath",
+        "The path to the account json file. The json file must be in the form of a json array, with each"
+            + "entry to be an account in its json form.")
+        .withRequiredArg()
+        .describedAs("account_json_file_path")
+        .ofType(String.class);
+
+    ArgumentAcceptingOptionSpec<String> zkServerOpt =
+        parser.accepts("zkServer", "The address of ZooKeeper server. This option is required.")
+            .withRequiredArg()
+            .describedAs("zk_server")
+            .ofType(String.class);
+
+    ArgumentAcceptingOptionSpec<String> storePathOpt = parser.accepts("storePath",
+        "The root path of helix property store in the ZooKeeper. Must start with /, and must not end "
+            + "with /. It is recommended to make root path in the form of /ambry/<clustername>/helixPropertyStore. This option is required.")
+        .withRequiredArg()
+        .describedAs("helix_store_path")
+        .ofType(String.class);
+
+    ArgumentAcceptingOptionSpec<Integer> zkConnectionTimeoutMsOpt = parser.accepts("zkConnectionTimeout",
+        "Optional timeout in millisecond for connecting to the ZooKeeper server. This option is not required, "
+            + "and the default value is 5000.")
+        .withRequiredArg()
+        .describedAs("zk_connection_timeout")
+        .ofType(Integer.class);
+
+    ArgumentAcceptingOptionSpec<Integer> zkSessionTimeoutMsOpt = parser.accepts("zkSessionTimeout",
```

Comment:
this is okay for now. But if you forsee the no of arguments growing in future, would prefer property File as input rather than parsing each arg.